duos-api/src/user/user.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Post, Body, Get, Param} from '@nestjs/common';
+import { UserDto } from 'src/Dtos/userDto';
+import { User } from 'src/models/user';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+
+    constructor(private userService: UserService){}
+
+    @Get(':id')
```

Comment:
(Future) Might be able to implement a custom decorator that returns the id that is attached to the http requests, it is attached when the request is validated, but this might be for daniels PR. Example in my night view repo under the file get-user.decorator.ts

Reply:
I'll see if I can just add the document id directly to the document so it's easier to reference